pages/get-involved.tsx

```diff
@@ -93,52 +92,69 @@ function BenefitsOfJoiningUs() {
   );
 }
 
-function Volunteer() {
+function PerfectFit() {
   return (
-    <>
-      <SectionContainer id="volunteer" maxWidth="md">
-        <Typography sx={{ my: 8 }} variant="h2">
-          {content.volunteerPage.header}
-        </Typography>
-        {content.volunteerPage.body.map((text) => (
-          <Typography
-            key={`volunteerText-${text}`}
-            variant="body1"
-            sx={{ my: 3 }}
-          >
-            {text}
-          </Typography>
-        ))}
-        <Grid container className="fact-card">
-          {content.volunteerPage.facts.map((fact) => (
-            <FactCard
-              key={fact.id}
-              details={fact.details}
-              icon={fact.icon}
-              border={false}
-            />
+    <SectionContainer id="perfect-fit" maxWidth="md">
+      <Typography variant="h2" alignItems="stretch">
+        {content.perfectFit.header}
+      </Typography>
+      <PaperSection sx={{
+        color: 'primary.contrastText', bgcolor: 'background.paper', px: 4, py: 1,
+      }}
+      >
+        <Typography color="text.primary">{content.perfectFit.body}</Typography>
+
+        <Grid container spacing={8} marginY={1}>
+          { content.perfectFit.perfectFitCards.map((card) => (
+            <Grid item sx={{ width: '100%', height: 'full' }} xs={12} md={6} key={`fit-card-${card.id}`} rowSpacing={2}>
+              <Box sx={{ backgroundColor: 'primary.dark', textAlign: 'center' }} height="100%" paddingLeft={1} borderRadius="20px">
+                <Typography fontWeight="bold" variant="subtitle1">{card.header}</Typography>
+                <Typography variant="body2" paddingBottom={1}>{card.body}</Typography>
+              </Box>
+            </Grid>
           ))}
+          <Grid item sx={{ width: '100%', height: 'full' }} xs={12} md={6} rowSpacing={2}>
```

Comment:
possible to center this image? especially on smaller screens when all of the boxes are in a single column